cmd/package_managers.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
is this something anyone uses in practice? Is it worth maintaining?

Reply:
No clue. Worth discussing this though. Maybe start a new issue for this?

I don't want to add breaking changes in this PR though, so will leave it as is for now.